src/main/java/uk/gov/companieshouse/pscfiling/api/controller/PscWithIdentificationFilingControllerImpl.java

```diff
@@ -56,7 +57,7 @@ public PscWithIdentificationFilingControllerImpl(final TransactionService transa
      */
     @Override
     @PostMapping(produces = {"application/json"}, consumes = {"application/json"})
-    public ResponseEntity<Object> createFiling(@PathVariable("transactionId") final String transId,
+    public ResponseEntity<PscCommunal> createFiling(@PathVariable("transactionId") final String transId,
```

Comment:
Is it possible to return `ResponseEntity<PscWithIdentificationFiling>` instead?

Reply:
amended and re run code ok